### PR TITLE
Task sources

### PR DESCRIPTION
**Pull request by keveri**
Rename DataSource to TaskSource.
Add configuration option for task sources.

### CODE REVIEW

**Comment by keveri** [comment]
Still needs a way to load the config from file.

**Comment by keveri** [comment]
If this is OK, Merge.

File: src/PeerReview/Types.hs [hunk 1]
```diff
@@ -53,11 +57,11 @@ data ReviewStatus = Waiting
                   | Accepted
                   deriving (Show)
 
-data ReviewTask = ReviewTask
-    { rtContent :: Text
-    , rtTaskId  :: TaskID
-    , rtUserId  :: UserID
-    } deriving (Show)
+data Task = Task
+    { tId      :: TaskID
```
Comment (otlaitil) [inline src/PeerReview/Types.hs]: @keveri I'd add one field here, usually a Task would have:

- task unique identifier (sha)
- task name ("exConcat")
- task user (email)
- task content (actual text-content)

What do you think?

**Comment by otlaitil** [comment]
Just one comment - otherwise looks good. I like the renaming of `DataSource` to `TaskSource`.

File: src/PeerReview/Types.hs (hunk 1 above)
Comment (keveri) [inline src/PeerReview/Types.hs]: Good idea. I think we can assume that tasks have name information.

Comment (otlaitil) [inline src/PeerReview/Types.hs]: Yeah, I think we can.. If we think of real life, each task has some title, identifier, or description to identify the task itself.

Another option would be to separate these two to `Task` and `TaskSubmission`

Comment (keveri) [inline src/PeerReview/Types.hs]: Good point.
Let's split Task into Task and Submission in a PR that will also split the TaskSource and define interfaces.

**Pull request merged**